analysis_driver/constants.py

```diff
@@ -24,6 +24,7 @@
 ELEMENT_NB_BASE_R2_CLEANED = 'clean_bases_r2'
 ELEMENT_NB_Q30_R1_CLEANED = 'clean_q30_bases_r1'
```

Comment:
Needs the latest reporting app pull request to be put into production before duplicate calculation can be used